core/templates/dev/head/base.html

```diff
@@ -205,7 +205,7 @@
                           </li>
                         {% else %}
                           <li class="dropdown oppia-clickable-dropdown pull-right">
-                            <a href="{{login_url}}" class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;">
+                            <a href="{{login_url}}" class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" ng-click="onLoginButtonClicked()">
```

Comment:
Because historically it didn't need one? The login url brings the user to a separate page, so this just functions like a normal html link.